src/main/res/values-ne/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
+<resources xmlns:tools="http://schemas.android.com/tools">
 	<string name="locRequestTitle">स्थान पहुँच</string>
-	<string name="locRequestMessage">यो अनुप्रयोगले विश्लेषण गर्दछ र तपाईंको क्षेत्रमा स्वास्थ्य परिणामहरू सुधार गर्न स्थान डाटा संकलन गर्दछ। स्थान पहुँचलाई अनुमति दिन "खोल्नुहोस्" चयन गर्नुहोस्, र त्यसपछि अर्को प्रॉम्प्ट पुष्टि गर्नुहोस्।</string>
+	<string name="locRequestMessage" tools:ignore="StringFormatInvalid">यो अनुप्रयोगले विश्लेषण गर्दछ र तपाईंको क्षेत्रमा स्वास्थ्य परिणामहरू सुधार गर्न स्थान डाटा संकलन गर्दछ। स्थान पहुँचलाई अनुमति दिन "खोल्नुहोस्" चयन गर्नुहोस्, र त्यसपछि अर्को प्रॉम्प्ट पुष्टि गर्नुहोस्।</string>
```

Comment:
The original translation has a placeholder `%s` to put the name of the branded app in the phrase:

```
<string name="locRequestMessage">%s collects location data to a...
```

But this Nepali translation replace the placeholder by I think the generic term "App" so the placeholder is not used. Android in that case raise an error with the new linter unless you add the ignore flag, because it is not a requirement to use all the placeholders.

BTW the translation was reviewed and actually fixed in the past by Sudip, so it was checked.

Reply:
Oh weird. So this ignore attribute **here** is necessary to handle an ignore in a string that overrides this?

Reply:
> this ignore attribute here is necessary to handle an ignore in a string that overrides this?
Got lost with you comment hahaha. Yes it is just to let the linter know that we acknowledge the lack of the placeholder but the translation works and it's fine.